core/domain/story_domain.py

```diff
@@ -560,9 +560,18 @@ def __init__(
         self.outline_is_finalized = outline_is_finalized
         self.exploration_id = exploration_id
         self.status = status
-        self.planned_publication_date = planned_publication_date
-        self.last_modified = last_modified
-        self.first_publication_date = first_publication_date
+        self.planned_publication_date_msecs = (
+            planned_publication_date_msecs
+            if planned_publication_date_msecs is not None else None
+        )
+        self.last_modified_msecs = (
+            last_modified_msecs
+            if last_modified_msecs is not None else None
+        )
+        self.first_publication_date_msecs = (
+            first_publication_date_msecs
+            if first_publication_date_msecs is not None else None
+        )
```

Comment:
Why do we need the "if X is not None else None" condition i.e. what is the value if the msec is not None?

Reply:
From the method signature, it looks like the value will be None if the float value is not provided.

Reply:
Thank you for pointing this out. I apologize for the oversight on my part.